t3kit/theme_t3kit
# [FEATURE] Add xing social icon

This Pull Request adds a XING icon to the social media icons content element.

![social_icons_with_xing](https://user-images.githubusercontent.com/7293310/30113998-be4da80e-9316-11e7-9c91-464f0b1c432a.png)

**Test Cases**
Frontend:
* A XING buttons should be available in the social media icons content element

Backend:
1. Edit a "Social Icons" content element in backend
2. You should see the input "XING: Pagelink" in the section "Additional settings"
3. Enter some link and save the changes
4. Refresh the frontend page and check if the XING button points to the URL you configured
